### PR TITLE
Pip freeze

### LINKED ISSUE #2347: All packages in requirements*.txt should have specified versions
We want to make sure that all packages defined in requirements files have exact specified versions and it's possible to install them on clean OSs.

### PR DESCRIPTION
**Pull request by maaktweluit**
Resolved #2347

- [x] Test how CI runs these new versions
- [x] Write doc on how / when to freeze

### CODE REVIEW

**Review by mfranciszkiewicz — APPROVED** [review]

**Comment by Krigpl** [comment]
dogscience
I'll let @etam take this :)

File: requirements-win_to-freeze.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+--extra-index-url https://builds.golem.network
+pyreadline
+pyvbox
+pywin32==221
```
Comment (etam) [inline requirements-win_to-freeze.txt:4]: why the version number?

Comment (maaktweluit) [reply requirements-win_to-freeze.txt:4]: To match the `easy_install`'ed hard-coded version in the build. To change this version a buildbot release is required.

Also want to reduce the scope of this issue by keeping the versions as close to the 0.14.0 release as possible.

**Comment by maaktweluit** [comment]
Related wiki page: https://github.com/golemfactory/golem/wiki/How-to-freeze-pip-modules

@etam replied to your review, please check

**Review by etam — APPROVED** [review]

**Comment by maaktweluit** [comment]
Branch got too old, creating a new one with the same steps executed, hoping to be merged before the next freeze

Sorry for the reviews!

**Pull request closed without merging**